lib/tasks/schools_data.rake

```diff
@@ -50,4 +51,48 @@ namespace :schools_data do
 
     puts "Done! created: #{created}, updated: #{updated}"
   end
+
+  desc "Generate data/lead_schools_publish.csv from Publish api"
+  task :generate_csv_from_publish, %i[output_path] => [:environment] do |_, args|
```

Comment:
`get_lead_schools_from_publish`? the description already says what it will create

Reply:
Done